docs/install_azure_flatcar/kind/master_install.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -xe
+
+# Assert that the current working directory is /root
+if [ "$(pwd)" != "/root" ]; then
+    echo "Error: The script must be run from the /root directory."
+    exit 1
+fi
+
+# create kubernetes cluster
+kind create cluster --config=/home/core/kind-config.yaml
```

Comment:
Using kind to create cluster and then install KubeEdge? Maybe https://github.com/kubeedge/kubeedge/blob/master/hack/local-up-kubeedge.sh has completed the similar work.

Reply:
@Shelley-BaoYue . This script install kubeedge : cloudcore and edgecore on the SAME machine and uses this approach: https://kubeedge.io/docs/setup/install-with-binary. My scripts provision 2 separate VMs and then install kubeedge using helm (for cloudcore) and keadm for edgecore. It helps to have an automated installation for this kind of use case of two VMs or one VM and one real edge device. Also it uses minimal size flatcar linux, which makes the install very fast.